test/test_merge.py

```diff
@@ -170,7 +170,8 @@ def test_nop_merge(self):
             a :-
                 from("alpine"),
                 (
-                    "1" = "1"
+                    "1" = "1",
+                    run("")
```

Comment:
The lack of `run` is intentional here - I'm testing that the merge code supports an empty list of operations. However if we do not allow it anymore I think it's better to just remove it...

Reply:
Actually on second thought, didn't you said that we will treat pure logic as valid layer expressions as well, to avoid the need for things like
```
a(X) :- X = "1", run("something"); X != "1", run("true").
```
?

Reply:
I only remember mentioning something like `logic, any_kind` should always be valid, but I don't remember suggesting that pure logic e.g. `logic, logic` should be a valid layer kind.
But I do actually run into this issue here https://github.com/modus-continens/openjdk-images-case-study/blob/f6069460b5c66118e779a285e8ccce102cc7e0fa/linux.Modusfile#L59.
Still I think it's actually worth the inconvenience of a no-op run like `run("")` because we currently can say for certain (I think) that a layer will produce more layers.

Reply:
So I guess I will remove this test